src/Protocol/ActivityPub/Processor.php

```diff
@@ -1343,6 +1345,29 @@ public static function storeReceivers(int $uriid, array $receivers)
 		}
 	}
 
+	private static function storeCapabilities(int $uriid, array $capabilities)
+	{
+		foreach (['pixelfed:canAnnounce' => Tag::CAN_ANNOUNCE, 'pixelfed:canLike' => Tag::CAN_LIKE, 'pixelfed:canReply' => Tag::CAN_REPLY] as $element => $type) {
+			if (!empty($capabilities[$element])) {
+				foreach ($capabilities[$element] as $capability) {
```

Comment:
You can reduce the indentation level by one like this:
```suggestion
			foreach ($capabilities[$element] ?? [] as $capability) {
```

Reply:
Done